sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/persons/EditPersonSteps.java

```diff
@@ -120,6 +122,38 @@ public EditPersonSteps(
           fillBirthName(newGeneratedPerson.getBirthName());
           fillNamesOfGuardians(newGeneratedPerson.getNameOfGuardians());
         });
+    Then(
+        "I click on See Cases for this Person button from Edit Person page",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(SEE_CASES_FOR_PERSON);
+          webDriverHelpers.waitForPageLoadingSpinnerToDisappear(150);
+        });
+    Then(
+        "I click on See CONTACTS for this Person button from Edit Person page",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(SEE_CONTACTS_FOR_PERSON);
+          webDriverHelpers.waitForPageLoadingSpinnerToDisappear(150);
+        });
+
+    Then(
+        "I click on Edit Case button from Cases card on Edit Person page",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(EDIT_CASES);
+          webDriverHelpers.waitForPageLoadingSpinnerToDisappear(150);
+        });
+    Then(
+        "I click on Edit Contact button from Contacts card on Edit Person page",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(EDIT_CONTACTS);
+          webDriverHelpers.waitForPageLoadingSpinnerToDisappear(150);
+        });
+    Then(
+        "I click on Edit Immunization button from Immunization card on Edit Person page",
```

Comment:
let's mention also that the action is performed for the immunization created via API, so we can differenciate it from other immunizations which can be created in UI flows in other steps/pages